presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSphericalGeoFunctions.java

```diff
@@ -144,4 +150,63 @@ private void assertDistance(String wkt, String otherWkt, Double expectedDistance
     {
         assertFunction(format("ST_Distance(to_spherical_geography(ST_GeometryFromText('%s')), to_spherical_geography(ST_GeometryFromText('%s')))", wkt, otherWkt), DOUBLE, expectedDistance);
     }
+
+    @Test
+    public void testArea()
+            throws IOException
+    {
+        //PA
+        assertPolygonAreaWithinPrecision("-79.76278 42.252649, -79.76278 42.000709, -75.35932 42.000709, -75.249781 41.863786, -75.173104 41.869263, -75.052611 41.754247, -75.074519 41.60637, -74.89378 41.436584, -74.740426 41.431108, -74.69661 41.359907, -74.828057 41.288707, -74.882826 41.179168, -75.134765 40.971045, -75.052611 40.866983, -75.205966 40.691721, -75.195012 40.576705, -75.069042 40.543843, -75.058088 40.417874, -74.773287 40.215227, -74.82258 40.127596, -75.129289 39.963288, -75.145719 39.88661, -75.414089 39.804456, -75.616736 39.831841, -75.786521 39.722302, -79.477979 39.722302, -80.518598 39.722302, -80.518598 40.636951, -80.518598 41.978802, -80.518598 41.978802, -80.332382 42.033571, -79.76278 42.269079, -79.76278 42.252649", 117255.1312637E6, 2E2);
+
+        //A polygon around the North Pole
+        assertPolygonAreaWithinPrecision("-135 85, -45 85, 45 85, 135 85, -135 85", 619002051089.943, 1E3);
```

Comment:
Let's rename this method to `assertArea` and remove precision parameter by hardcoding 0.01 precision. Let's also pass full WKT so that we can have just one assert*** method.

```
    assertArea("POLYGON ((-135 85, -45 85, 45 85, 135 85, -135 85))", 619_002_051_089.943);

    private void assertArea(String wkt, double expectedArea)
    {
        assertFunction(format("ABS(ROUND((ST_Area(to_spherical_geography(ST_GeometryFromText('%s'))) / %s - 1 ) * %s, 0))", wkt, expectedArea, 0.01), DOUBLE, 0.0);
    }
```